src/process_utility.c

```diff
@@ -365,8 +365,22 @@ process_copy(ProcessUtilityArgs *args)
 	Cache *hcache;
 	Oid relid;
 
+	/* We only copy for COPY FROM. Since hypertables do not contain any data,
+	 * we skip the copy for COPY TO, but print an informative message. */
 	if (!stmt->is_from || NULL == stmt->relation)
+	{
+		if (stmt->relation)
+			ereport(NOTICE,
+					(errmsg("hypertables has no data in main table, no data will be copied"),
+					 errdetail("Data for hypertables are not stored in the main table, it is "
```

Comment:
Suggestions:
> Since data are stored in chunks, COPY TO on hypertable itself produces no result.

Reply:
See above.

Reply:
I agree with @k-rus that this error is too much in the weeds. Either skip or simplify to something like:

```
A hypertable is partitioned into chunks that hold the data and therefore does not support COPY TO.
```